sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -662,6 +666,10 @@
                 <title>Azure Text Analytics</title>
                 <packages>com.azure.ai.textanalytics*</packages>
               </group>
+              <group>
+                  <title>Azure Management</title>
+                  <packages>com.azure.management*</packages>
```

Comment:
Is this correct?

Reply:
It's what was originally in their parent so I'm assuming so.